### DISCUSSION TITLE
Only open-gl is avalible to select in renderer menu?

### DISCUSSION THREAD

**Discussion by chamychu** [ppy/osu#23117] [Q&A] [answered]
I was trying to select vulkan as the renderer but only open-gl and open-gl-legacy appear.
 is vulkan not yet added on linux?

![Screenshot from 2023-04-03 17-04-40](https://user-images.githubusercontent.com/107549591/229533588-c8d4858f-a179-482e-992a-d4883930b3e2.png)

heres my system info:
https://pastebin.com/kWpqVhgj

**Comment by Rekunan** [comment, marked as answer]
Vulkan got pulled due to being too unstable #22961

**Reply by chamychu** [reply]
will it be added back anytime soon?

**Reply by Rekunan** [reply]
When it's more stable or when they can make it more stable yes.
